src/NuGet.Core/NuGet.Packaging/Signing/TrustStore/X509TrustStore.cs

```diff
@@ -71,7 +71,7 @@ internal static IX509ChainFactory CreateX509ChainFactoryForDotNetSdk(ILogger log
                 if (SystemCertificateBundleX509ChainFactory.TryCreate(
                     out SystemCertificateBundleX509ChainFactory systemBundleFactory))
                 {
-                    logger.LogVerbose(
+                    logger.LogInformation(
```

Comment:
Sorry, I didn't read the descriptions in the PR.

Reply:
From the [PR's description](https://github.com/NuGet/NuGet.Client/pull/4720#issue-1302291237):

> Also, the verbosity level of log messages indicating which X.509 trust store will be used has been changed from verbose to informational, because in the `dotnet add package` scenario, verbose messages aren't displayed, informational messages are, and there's no obvious way for a user to increase verbosity level.